spring-web/src/test/java/org/springframework/web/multipart/support/StandardMultipartHttpServletRequestTests.java

```diff
@@ -84,11 +84,13 @@ public void multipartFileResource() throws IOException {
 		new FormHttpMessageConverter().write(map, null, output);
 
 		assertThat(output.getBodyAsString(StandardCharsets.UTF_8)).contains(
-				"Content-Disposition: form-data; name=\"file\"; filename=\"myFile.txt\"\r\n" +
-						"Content-Type: text/plain\r\n" +
-						"Content-Length: 6\r\n" +
-						"\r\n" +
-						"myBody\r\n");
+				"""
+						Content-Disposition: form-data; name="file"; filename="myFile.txt"\r
+						Content-Type: text/plain\r
+						Content-Length: 6\r
+						\r
+						myBody\r
+						""");
```

Comment:
Instead of manually appending `\r` to every line in a text block, we prefer to write text blocks as "plain text" and then adjust the line endings only if necessary.

For example, in this particular case we would remove all `\r` entries and apply `.replace("\n", "\r\n")` to the text block.

NO need to make these changes in this PR though. I have already applied such changes locally which will be pushed to `main` along with your PR.